projects/aca-folder-rules/assets/i18n/en.json

```diff
@@ -31,6 +32,13 @@
         "UPDATE": "Items are updated",
         "OUTBOUND": "Items are deleted or leave this folder"
       },
+      "OPTIONS": {
+        "CASCADE":  "Rule applies to subfolders",
+        "ASYNCHRONOUS": "Run rule in the background",
+        "ENABLED": "Disable rule",
```

Comment:
This confused me for a second. It is the right way around since it _is_ enabled and it's the checkbox to disable it, but I would clarify this by changing the tag name simply to "DISABLE_RULE".

Reply:
I will change in the next PR